docs/dev/agent_integration.md

```diff
@@ -21,6 +22,52 @@ The required Datadog Agent integration development tools include:
 - [Docker][4] to run the full test suite.
 - The git [command-line][5] or [GitHub desktop client][19].
 
+You can list an Agent integration for a price in the Datadog Marketplace, or list it out-of-the-box on the Integrations page.
+
+{{< tabs >}}
+{{% tab "Marketplace Integrations" %}}
+## Set up a directory and clone the Marketplace repository
+
+Set up a directory:
+
+1. Request access to the [Marketplace repository][101] by following the instructions in the [Marketplace documentation][102].
+2. Create a `dd` directory:
+   {{< code-block lang="shell" >}}mkdir $HOME/dd{{< /code-block >}}
+
+   The Datadog Development Toolkit command expects you to be working in the `$HOME/dd/` directory. This is not mandatory, but working in a different directory requires additional configuration steps.
+3. Once you have been granted access to the Marketplace repository, create the `dd` directory and clone the `marketplace` repo:
+   {{< code-block lang="shell" >}}git clone git@github.com:DataDog/marketplace.git{{< /code-block >}}
+4. Create a feature branch to work in.
+    {{< code-block lang="shell" >}}git switch -c <YOUR INTEGRATION NAME> origin/master{{< /code-block >}}
+
+## Install and configure the Datadog development toolkit
+
+The Agent Integration Developer Tool allows you to create scaffolding when you are developing an integration by generating a skeleton of your integration tile's assets and metadata. For instructions on installing the tool, see [Install the Datadog Agent Integration Developer Tool][103].
+
+After you install the Developer tool, configure it for the `marketplace` repo:
+
+Set `marketplace` as the default working repository:
```

Comment:
Maybe this can be combined into one line

Reply:
Going to match it to ootb integrations directions. Numbered set and set working repo

Reply:
The reason it isn't one line is because a partner may have set the repo in a directory other than home. @anawishnoff I agree this does look weird. We can just say "After you install the Developer tool, configure it for the `marketplace` repo by following the instructions below"

Reply:
That sounds good